js/views/itemEditVw.js

```diff
@@ -217,19 +220,33 @@ module.exports = Backbone.View.extend({
   },
 
   deleteImage: function(e) {
-    var imgIndex = $(e.target).closest('.itemImg').data('index');
+    var imageUploadArray, imgIndex = $(e.target).closest('.itemImg').data('index');
```

Comment:
It is a bad practice IMO because not only of hoisting, but as I've already seen a couple times in the codebase, it can allow for the unawareness of duplicating the declaration, as well as when it is var'd again it will clobber the previous value. 
With strict mode, this is not possible.